build.gradle.kts

```diff
@@ -309,6 +311,17 @@ tasks.register<Install>("installAll") {
     installDirPropertyName = ::gradle_installPath.name
 }
 
+val allIncubationReports = tasks.register<IncubatingApiAggregateReportTask>("allIncubationReports") {
+    val allReports = subprojects.flatMap { it.tasks.withType(IncubatingApiReportTask::class) }
+    dependsOn(allReports)
+    reports = allReports.associateBy({ it.title.get()}) { it.textReportFile.asFile.get() }
+}
+tasks.register<Zip>("allIncubationReportsZip") {
+    destinationDir = file("$buildDir/reports/incubation")
+    baseName = "incubating-apis"
+    from(allIncubationReports.get().htmlReportFile)
+}
+
```

Comment:
Is it worth zipping it up? My expectation would be that the report _should_ be fairly small once we get rid of the huge mass of outdated incubations.

Reply:
The report is 70kB, so maybe not worth it. I tend to zip everything, because, you know, my bandwidth :D